scipy/sparse/linalg/_isolve/tests/test_symmlq.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from numpy.testing import assert_equal, assert_
+from scipy.sparse import csr_matrix
+from scipy.sparse.linalg import LinearOperator
+from scipy.sparse.linalg._isolve import symmlq
+
+
+def get_sample_problem():
```

Comment:
This is not sufficient for a sparse function test. You need to provide a few sparse problems and not the toy dense problems. https://sparse.tamu.edu/ has thousands of examples of different kinds of sparse examples. Please thoroughly test the algorithms for any kind of input data and type.